react/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+    "plugins": ["babel-plugin-styled-components"]
+}
```

Comment:
I recommend putting this in a "babel" key in package.json so the subproject root doesn't get cluttered with dotfiles. See https://babeljs.io/docs/en/configuration#packagejson